react/components/Footer/global.css

```diff
@@ -105,10 +105,6 @@
     margin-top: 0;
```

Comment:
Tachyons class ```mt0```.

Use the [Tachyons](https://github.com/vtex/vtex-tachyons/blob/master/tachyons.css) class always as possible.